storage/filestore.go

```diff
@@ -61,6 +67,71 @@ type FilesystemStore struct {
 	perms   os.FileMode
 }
 
+func (f *FilesystemStore) moveKeyTo0Dot4Location(file string) {
+	keyID := filepath.Base(file)
+	fileDir := filepath.Dir(file)
+	d, _ := f.Get(file)
+	block, _ := pem.Decode(d)
+	if block == nil {
+		logrus.Warn("Key data for ", file, " may have been tampered with/ is invalid. The key has not been migrated and may not be available")
```

Comment:
super nit : can we mention how we couldn't decode the PEM or filestructure?  Something like `Key data for <file> could not be decoded as a valid PEM block.  The key will not been migrated and may not be available`